src/main/io/gps.c

```diff
@@ -734,6 +734,8 @@ static bool gpsNewFrameNMEA(char c)
                             break;
                         case 9:
                             gps_Msg.altitude = grab_fields(string, 0);     // altitude in meters added by Mis
+                            if (string[0] == '-')
```

Comment:
modifying  `grab_fields` to (conditionally) support negative values will be cleaner fix

Reply:
@ledvinap Please check if the modified grab_fields() may work.
Another potential problem: NMEA altitude seems to be converted to 1m/lsb which does match the ublox altitude conversion to 1cm/lsb. Should we care about this?